clients/csharp-dotnet-core/voice-assistant-test/README.md

```diff
@@ -105,6 +105,9 @@ Here is the full list:
 
 <font color="red">TODO: Make the above relative path to InputFolder?</font>
 
+>#### UseKeywordVerification

```

Comment:
Looking at other boolean properties, looks like you can simply call it "KeywordVerification" (no need for the "Use"). Or "KeywordVerificationEnabled"

Reply:
I like KeywordVerificationEnabled :)